# Generated for NaoQi method ALSpeechRecognition.setParameter
# Sets a parameter of the speech recognition engine. Note that when the ASR engine language is set to Chinese, no parameter can be set.               The parameters that can be set and the corresponding values are:               "EarUseSpeechDetector" - Values : 0 (No), 1 (Regular), or 2 (Cepstral) : type of speech detector used by the ASR engine - default value is 2.                "EarSpeed" - Values : 0 to 3 - 0 is slowest and most accurate. 3 is the fastest but may add some recognition errors, especially if you talk fast because some parts of the audio data is not processed. - default value is 2.               "EarUseFilter" : Values : 0 (no) or 1 (yes) - Applies a High-Pass filter on the input signal - default value is 0.

# Name of the parameter.
string paramName
# Value of the parameter.
float32 paramValue
---


